src/progress-bar/types.js

```diff
@@ -25,6 +26,8 @@ export type ProgressBarPropsT = {
   successValue: number,
   /** If set to false, label is hidden and `getProgressLabel` is ignored. */
   showLabel: boolean,
+  /** May be renamed to variant: string */
```

Comment:
what does this comment mean? can it be improved to describe what this prop does?